app/models/planning_application.rb

```diff
@@ -482,6 +478,36 @@ def create_audit!
     audit_created!(activity_type: "created", activity_information: Current.api_user&.name || Current.user&.name)
   end
 
+  def audit_updated!
+    if saved_changes?
+      saved_changes.keys.intersection(PLANNING_APPLICATION_PERMITTED_KEYS).map do |attribute_name|
+        next if saved_change_to_attribute(attribute_name).all?(&:blank?)
```

Comment:
I think we should add a spec for this. Maybe just a simple system spec will do . There wasn't any coverage before so would be good to add something now